rest/blip_sync.go

```diff
@@ -14,11 +14,16 @@ import (
 
 // HTTP handler for incoming BLIP sync WebSocket request (/db/_blipsync)
 func (h *handler) handleBLIPSync() error {
-
 	h.db.DatabaseContext.DbStats.StatsDatabase().Add(base.StatKeyNumReplicationsActive, 1)
 	h.db.DatabaseContext.DbStats.StatsDatabase().Add(base.StatKeyNumReplicationsTotal, 1)
 	defer h.db.DatabaseContext.DbStats.StatsDatabase().Add(base.StatKeyNumReplicationsActive, -1)
 
+	// Exits early when the connection can't be switched to websocket protocol.
+	if _, ok := h.response.(http.Hijacker); !ok {
+		base.DebugfCtx(h.db.Ctx, base.KeyHTTP, "Non-upgradable request received for BLIP+WebSocket protocol")
+		return base.HTTPErrorf(http.StatusForbidden, "Can't upgrade this request to websocket connection")
```

Comment:
Use `426 Upgrade Required` status code

https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/426

```suggestion
		return base.HTTPErrorf(http.StatusUpgradeRequired, "Can't upgrade this request to websocket connection")
```

Reply:
Sure, StatusForbidden has been replaced with StatusUpgradeRequired